themes/theme-gmd/themeApi/ProductCard/components/Render/index.jsx

```diff
@@ -11,6 +11,7 @@ import {
 } from '@shopgate/engage/product';
 import Link from '@shopgate/pwa-common/components/Link';
 import RatingStars from '@shopgate/pwa-ui-shared/RatingStars';
+import { useWidgetSettings } from '../../../../../../libraries/engage/core/hooks/useWidgetSettings';
```

Comment:
please use '@shopgate/engage/core' to import 'useWidgetSettings', because there's an index file which exports among other things all hooks.

Reply:
relative imports from theme level to libraries don't work in production. they only work locally. In prod all modules in /libraries are published to npm and the themes are using them as node_modules